package.json

```diff
@@ -51,5 +55,11 @@
   "peerDependencies": {
     "react": "^16.8.0",
     "react-native": "^0.59.0"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "npm test",
```

Comment:
Can we remove this `pre-commit` hook? I think having the `pre-push` is enough, I use commit a lot, it could be annoying for people like me. WDYT?

Reply:
I was testing whether the hooks are working or not, please dont review. And yes, i'll be removing the pre-commit hook. Its very frequent for me as well, also this hook doesnt work when you commit through vscode default git plugin.

Reply:
The pre-commit hook was something lint-staged added, and i was trying to see its working. Even I dont like having pre-commit hooks, because I commit to often. Will be removing this.